src/model/kaniCommand.ts

```diff
@@ -1,4 +1,45 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+import { execFile } from "child_process";
+import * as fs from 'fs';
+import * as path from 'path';
+
 export interface CommandArgs {
     command: 'kani' | 'cargo kani';
     args: string[];
 }
+
+// Disable the "Shell" option
+export const options = {
+	shell: false,
+};
+
+// Get the path to the cargo command
+export function getKaniPath(): Promise<any>{
+    return new Promise((resolve, reject) => {
+        execFile('which', ['kani'], (error, stdout, stderr) => {
+            if (error) {
+              console.error(`execFile error: ${error}`);
```

Comment:
Suggest a more actionable error message, such as `Could not find kani binary on $PATH`?